challenge.rb

```diff
@@ -1,26 +1,27 @@
 
-def capitalize_each_string(input)
+def capitalize_each_string(*animal)
   #implement your solution here
+  animal.each { |a| puts a.capitalize }
```

Comment:
Hey Nicolas,
I've revised the challenge.rb file so that all test return "True".  
Q: Since the map method presumably changed the array to all caps, why didn't we need to downcase the array prior to making the other tests pass?
Q: What's the difference between the find and find_all methods?  I was surprised that the find method did not return True.

Thanks for your help!